test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1

```diff
@@ -13,7 +13,7 @@ Describe 'Clipboard cmdlet tests' -Tag CI {
         }
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-linux`
Get-Process.Test for process property = Name
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   15
Strings differ at index 0.
Expected: 'pwsh'
But was:  'ConsoleHost mai'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 74
74:         (Get-Process -Id $PID).Name | Should -BeExactly "pwsh"
```
Process Parent property.Has Parent process property
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 128
128:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent' | Should -Not -BeNullOrEmpty
```
Process Parent property.Has valid parent process ID property
```powershell
Expected 7780, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 133
133:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent.Id' | Should -Be $PID
```